src/content/getting-started/network-policies/index.md

```diff
@@ -110,6 +110,31 @@ Note that the namespace needs to exist before you apply the NetworkPolicy to it.
 
 The default policy shown above will limit ingress and egress traffic in the namespace applied. You can also restrict only for `egress` or `ingress`.
 
+## DNS
+
+In case you want to limit the DNS access within the cluster or namespace, you can use a NetworkPolicy that targets just DNS like:
```

Comment:
```suggestion
In case you want to limit pods to only DNS access within the cluster or namespace, you can use a NetworkPolicy that targets just DNS like:
```

Reply:
I think this need to be phrased a little bit differently. Generally speaking, if you want to limit egress to anything specific, allowing DNS is often an implicit requirement that you need to explicitly allow for it to work.
Hence, this is not only about limiting DNS but also about not forgetting to grant DNS where required.
As an example, I came here after first looking at [this](https://github.com/ahmetb/kubernetes-network-policy-recipes/blob/master/11-deny-egress-traffic-from-an-application.md#allowing-dns-traffic) from the somewhat official Kubernetes documentation, hoping to figure out why that example was no working for me at all.